coil/README.md

```diff
@@ -93,6 +93,24 @@ CoilImage(
 
 Accompanist Coil supports GIFs through Coil's own GIF support. Follow the [setup instructions](https://coil-kt.github.io/coil/gifs/) and it should just work.
 
+## Custom ImageLoader
+
+If you wish to provide a default [`ImageLoader`](https://coil-kt.github.io/coil/image_loaders/) to use across all of your `CoilImage`
+calls, we provide the `AmbientImageLoader` ambient. You can see it like so:
```

Comment:
Link to ambient docs? https://developer.android.com/reference/kotlin/androidx/compose/runtime/Ambient.html

Reply:
Added in https://github.com/chrisbanes/accompanist/commit/f2112d6270999e78c9748c83c7ead8d7ee0acc23